appengine/endpoints/README.md

```diff
@@ -1,7 +1,9 @@
 # Google Cloud Endpoints & App Engine Flexible Environment & Ruby
 
 This sample demonstrates how to use Google Cloud Endpoints on
-Google App Engine flexible environment using Ruby.
+Google App Engine flexible environment using Ruby. This sample requires that you
+have [Ruby](https://www.ruby-lang.org/en/documentation/installation/) 2.0.0 or
```

Comment:
This blew up when using 1.9.3, which was installed by default on my Ubuntu computer.